FWCore/Services/plugins/SimpleMemoryCheck.cc

```diff
@@ -402,6 +505,14 @@ namespace edm {
       //       &SimpleMemoryCheck::preEventProcessing);
       //  iReg.watchPreModule(this,
       //       &SimpleMemoryCheck::preModule);
+
+      if (showJemallocInfo_) {
```

Comment:
So `showJemallocInfo_ can't be used with `sampleEveryNSeconds_`?

Reply:
Presently the jemalloc information is not shown on any "periodic" printouts. It is shown only in earlyTermination printout (that is added in this PR), and in the endJob for the state of the endJob, and for the peak RSS and VSIZE recordings.

Written that, it would be easy to change the jemalloc information to be shown on all "periodic" printouts too. I opted not to (at this time) to minimize the number of calls to `mallctl()`.

Reply:
(For me) Describe the behavior of `showJemallocInfo` parameter in the `fillDescriptions()`.